cve_bin_tool/checkers/webkitgtk.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
```

Comment:
```suggestion
# Copyright (C) 2021 Intel Corporation
# SPDX-License-Identifier: GPL-3.0-or-later

```

We don't need the shebang line (it's a helper for people executing the file directly so they would get the right interpreter, but there's no good reason to execute an invidual checker that way because it wouldn't work.) but I do want the SPX licensing and copyright header on all files now to comply with open source best practices.

Reply:
Made the changes